### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3` (v1.12.0.dev0)

### CODE REVIEW

File: brayns/io/MorphologyLoader.cpp [hunk 1]
```diff
@@ -455,7 +455,7 @@ class MorphologyLoader::Impl
      * not apply
      * @param compartmentReport Compartment report to map to the morphology
      * @param scene Scene to which the morphology should be loaded into
-     * @return True if the loading was successfull, false otherwise
+     * @return True if the loading was successful, false otherwise
```
Comment (rolandjitsu) [inline brayns/io/MorphologyLoader.cpp]: [successful](http://www.thesaurus.com/browse/successful) is the valid form of the word. Also check [successfull-successful-is-this-a-uk-us-difference](https://english.stackexchange.com/a/4619).

Comment (luzpaz) [reply brayns/io/MorphologyLoader.cpp]: @rolandjitsu I'm a little confused. The 2nd link says:
> I would say successfull is a typo. Neither Merriam-Webster nor Wiktionary mention it as an alternative spelling. The British National Corpus has 10695 cites for successful and exactly one for successfull. According to OneLook, 33 dictionaries have an entry for successful, but only Wordnik has a few cites for successfull (without a definition).

Comment (rolandjitsu) [reply brayns/io/MorphologyLoader.cpp]: If you check the first two links mentioned in the answer, none have a definition for `successfull`.  And the entry `successful` is mentioned in 33+ dictionaries. I guess that means the the proper form of this word should be `successful`.

But the answer notes if we use the adverb, then we use two `l` characters: `successfully`.

Comment (luzpaz) [reply brayns/io/MorphologyLoader.cpp]: exactly, so my revision here is valid.

Comment (rolandjitsu) [reply brayns/io/MorphologyLoader.cpp]: Haha, yes. Sorry, I got confused for a second there, I saw `successful` -> `successfull` not the other way around.

**Comment by rolandjitsu** [comment]
@tribal-tec the checks seem to be hanging for some reason, is this because of the failing builds on Jenkins?

**Comment by rdumusc** [comment]
@rolandjitsu if bbpbuildbot says "Can one of the admins verify this patch?", someone needs to say "test this please" ;-)

**Comment by rolandjitsu** [comment]
@rdumusc thanks for the info 🍻

File: plugins/engines/ospray/ispc/render/SimulationRenderer.ispc [hunk 2]
```diff
@@ -505,7 +505,7 @@ inline vec3f SimulationRenderer_shadeRay(
 
             if (attributes.opacity > 0.01f && moreRebounds)
             {
-                // Compute indirect lightening contribution
+                // Compute indirect lightning contribution
```
Comment (tribal-tec) [inline plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: it's even 'lighting'. No thunderstorms here :)

Comment (luzpaz) [reply plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: oops. yea..so keep this `lightening` right ?

Comment (luzpaz) [reply plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: hehe...yea..it is `lighting`! Ok, fixed

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
@luzpaz feel free to merge, and thanks for the contribution! :)

**Pull request merged**